### PR TITLE
Add RTL support (closes #257 and closes #347)

### LINKED ISSUE #257: Add option to set direction of message to RTL

### PR DESCRIPTION
**Pull request by ricardobrandao**

### CODE REVIEW

**Comment by ricardobrandao** [comment]
Not sure what people thinks about inverting the dialog box position for RTL (i.e. right goes left and the opposite). If _everyone_ agrees I can also add those changes.

**Comment by TimFerrell** [comment]
Cool work! Glad to see this finally getting implemented. Some feedback:

- There are no tests. I recognize that the JS footprint for this is very small, but please add some to check for the presence of the RTL setting. Bonus points for validating the CSS.
- Make text in the demo "Right-To-Left" instead of "Is Right-To-Left"
- I would also like to see an example of this using an RTL language. Can you please attach an image of this functionality using an RTL font?

**Comment by ricardobrandao** [comment]
@TimFerrell thanks! Glad to help 👍 

I've fixed the demo and added some tests. 

Also, here it is the example:
toastr-rtl

File: build/toastr.css [hunk 1]
```diff
@@ -178,24 +188,41 @@ button.toast-close-button {
...
+  #toast-container > div.rtl {
+    padding: 8px 50px 8px 8px;
+  }
   #toast-container .toast-close-button {
     right: -0.2em;
     top: -0.2em;
   }
+  #toast-container .rtl .toast-close-button {
+    left: -0.2em;
+    right: 0.2em;
```
Comment (TimFerrell) [inline build/toastr.css]: Should top?

File: build/toastr.css [hunk 2]
```diff
@@ -178,24 +188,41 @@ button.toast-close-button {
...
   }
+  #toast-container > div.rtl {
+    padding: 8px 50px 8px 8px;
+  }
   #toast-container .toast-close-button {
     right: -0.2em;
     top: -0.2em;
   }
+  #toast-container .rtl .toast-close-button {
+    left: -0.2em;
```
Comment (TimFerrell) [inline build/toastr.css]: Should be top.

File: build/toastr.css (hunk 1 above)
Comment (ricardobrandao) [inline build/toastr.css]: Already tops from the [non-rtl version](https://github.com/CodeSeven/toastr/pull/439/files#diff-cc044eb2261831f76b5653c793d7ecd8R196).

**Comment by TimFerrell** [comment]
Thanks very much for the work!

**Comment by ricardobrandao** [comment]
@TimFerrell np :)

Are there any chance of creating a release for this? We use jspm in our project and it would be great not to point directly to the **develop** branch.

**Comment by TimFerrell** [comment]
Let me take a look over what changes we've got in develop and what's in the development pipeline. I think we've got enough in the diff to justify a new release to master, just want to confirm.

**Pull request merged**